content/metadata.yaml

```diff
@@ -1,10 +1,10 @@
 ---
-title: "Meta Review"
+title: "Open, collaborative writing with Manubot"
 keywords:
   - collaborative review
   - continuous integration
   - publishing
-  - manubot
+  - Manubot
```

Comment:
I don't think keyword capitalization matters, hence the all lowercase convention. Hoewver, no strong feeling here.

Reply:
I capitalized it as a proper noun, but no feelings here either.

Reply:
Proper noun. I feel strongly in favor of capitalization 😆